cmake/Modules/FindChipmunk.cmake

```diff
@@ -60,6 +60,24 @@ FIND_LIBRARY(CHIPMUNK_LIBRARY
 
 set(CHIPMUNK_INCLUDE_DIRS "${CHIPMUNK_INCLUDE_DIR}")
 
+if(OHOS)
+  # 主动set路径
```

Comment:
Please don't use Chinese. Please fix it in all files.

Reply:
fix